monix-tail/shared/src/main/scala/monix/tail/internal/IterantTakeEveryNth.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2014-2018 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.tail.internal
+
+import monix.tail.Iterant.{Halt, Last, Next, NextBatch, NextCursor, Suspend}
+import monix.tail.Iterant
+import cats.syntax.all._
+import cats.effect.Sync
+import monix.execution.misc.NonFatal
+import monix.tail.batches.BatchCursor
+import monix.tail.internal.IterantUtils._
+
+import scala.collection.mutable.ArrayBuffer
+
+private[tail] object IterantTakeEveryNth {
+  /**
+    * Implementation for `Iterant#takeEveryNth`
+    */
+  def apply[F[_], A](source: Iterant[F, A], n: Int)
+                    (implicit F: Sync[F]): Iterant[F, A] = {
```

Comment:
Don't like this indentation style and the Monix code is pretty consistent on not doing things dangling to the right like that 😀 personal preference, but it's good to be consistent.

Put this signature all on one line, delete the empty line between it and `require` below and you're done 😀

Reply:
Consistency is the king! Fixed. It's my first PR here, need to learn codebase more. Thanks for the fast and thorough review!